roles/sosreport/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: sosreport - install required packages
+  become: true
+  ansible.builtin.package:
+    name: "{{ item }}"
+    state: present
+  loop: "{{ required_packages_sosreport }}"
+
+- name: sosreport - remove temporary directory
+  become: true
+  ansible.builtin.file:
+    path: "{{ sosreport_tmpdir }}"
+    state: absent
+
+- name: sosreport - create required directories
+  become: true
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ operator_user }}"
+    group: "{{ operator_group }}"
```

Comment:
Add ``operator_user`` + ``operator_group`` to the defaults.

Reply:
Done